notebook.ipynb

```diff
@@ -2,9 +2,17 @@
  "cells": [
```

Comment:
<p>It would be helpful if you also briefly repeated here what the metrics mentioned in the paper represent/how they should be interpreted.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/reproduce-deep-prior-4Dvar/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>I'll open an issue on this to someone from the EDS book or other communities can describe better the metrics. Thanks for the suggestion.</p>